pkg/apis/minio.min.io/v1/conversion.go

```diff
@@ -84,6 +83,10 @@ func (src *Tenant) ConvertTo(dstRaw conversion.Hub) error {
 	dst.Spec.ImagePullPolicy = src.Spec.ImagePullPolicy
 	dst.Spec.SideCars = src.Spec.SideCars
 	dst.Spec.ExposeServices = src.Spec.ExposeServices
+	// Apply the securityContext to all the Pools
+	for _, p := range dst.Spec.Pools {
```

Comment:
is the security context per pool on v1?

Reply:
No, it is per pool on V2. So while conversion I am setting the SecurityContext for all the pools